### ISSUE TITLE
Network capture not working in main

### ISSUE THREAD

**Issue by yanivagman** [aquasecurity/tracee#947]
Seems that we have a regression with network capture feature since we bumped the libbpf version (to support external btf files).
```
sudo ./dist/tracee-ebpf --capture net=wlp2s0 -t e=open --capture clear-dir
libbpf: Kernel error message: Exclusivity flag on, cannot modify
2021/08/23 13:33:50 error creating Tracee: tc hook create: file exists
```

**Comment by yanivagman** [comment]
After bisecting libbpf, I found the offending commit:
commit a1bd8104a9f1c1a5b9cd0f698c886296749a0ce9 (HEAD)
```
Author: Kumar Kartikeya Dwivedi <memxor@gmail.com>
Date:   Sat Jun 12 08:05:01 2021 +0530

    libbpf: Set NLM_F_EXCL when creating qdisc
    
    This got lost during the refactoring across versions. We always use
    NLM_F_EXCL when creating some TC object, so reflect what the function
    says and set the flag.
    
    Fixes: 715c5ce454a6 ("libbpf: Add low level TC-BPF management API")
    Signed-off-by: Kumar Kartikeya Dwivedi <memxor@gmail.com>
    Signed-off-by: Daniel Borkmann <daniel@iogearbox.net>
    Link: https://lore.kernel.org/bpf/20210612023502.1283837-3-memxor@gmail.com

diff --git a/src/netlink.c b/src/netlink.c
index efbb50a..cf9381f 100644
--- a/src/netlink.c
+++ b/src/netlink.c
@@ -457,7 +457,7 @@ static int tc_qdisc_modify(struct bpf_tc_hook *hook, int cmd, int flags)
 
 static int tc_qdisc_create_excl(struct bpf_tc_hook *hook)
 {
-       return tc_qdisc_modify(hook, RTM_NEWQDISC, NLM_F_CREATE);
+       return tc_qdisc_modify(hook, RTM_NEWQDISC, NLM_F_CREATE | NLM_F_EXCL);
 }
```

Locally removing NLM_F_EXCL from 3rdparty/libbpf solves the problem.
I'm now trying to figure out what is causing this error when the flag is set.

**Comment by yanivagman** [comment]
Opened https://lore.kernel.org/bpf/CAMy7=ZXTiaX9xzNi5aOavwsf+mziJ=w-EcHH2f=cJmCGr3EPQA@mail.gmail.com/T/#u in the mailing list to check if this is the expected behavior or a bug in libbpf
